test/cmp/ok/neg-xyz-x.cmp.ok

```diff
@@ -1,7 +1,4 @@
-(unknown location): Compatibility error [M0169], stable variable y of previous type
+(unknown location): warning [M0169], stable variable y of previous type
   Int
-cannot be discarded; promote to type `Any` instead
-(unknown location): Compatibility error [M0169], stable variable z of previous type
-  Bool
-cannot be discarded; promote to type `Any` instead
-FALSE
+ will be discarded. This may cause data loss. Are you sure?
+TRUE
```

Comment:
Note that we've dropped the error for z entirely.... (though I'm curious why it was produced before...)

Reply:
Thanks for noticing. This is related to the bug above. Sorry that it slipped my attention. (I initially had no warning, just allowed it, and then later introduced the warning, but half-baked.)